libcrux-ml-dsa/src/simd/avx2/generate_uniform_rej_sample_table.py

```diff
@@ -0,0 +1,36 @@
+#! /usr/bin/env python3
+
+
+def trailing_zeros(num):
+    return (num & -num).bit_length() - 1
+
+
+with open("uniform_rej_sample_table.rs", "w+") as f:
```

Comment:
Could you write this code in Rust? If you do, we could verify it....

Reply:
I added an executable specification to `libcrux-ml-dsa/src/simd/avx2/uniform_rej_sample_table.rs` and a unit-test in there. Could you take a look and let me know?